docs/source/classroom/cps_ood.md

```diff
@@ -2,120 +2,90 @@
 :::
 
-## Submit CPS class assignments
+## Submit Class Assignments
 
 :::{important}
-Due to the write-only access permissions on the **assignments** directory, it is required to use the command line interface (Linux Terminal) to submit assignments. **Using other methods, such as the JupyterLab interface or OOD File Explorer, currently does not work**.
+Due to the write-only access permissions on the **assignments** directory, it is required to use the command line interface (Linux Terminal) to submit assignments. **Using other methods, such as the JupyterLab interface or OOD file explorer, currently does not work**.
 :::
 
-To submit your assignment (for example, named: **Assignment1.ipynb**) to the **assignments** directory, open the JuypterLab New Launcher by clicking the **File** top menu option, and then selecting **New Launcher**:
-
-![Alt text](../images/cps-ood-jupyterlab-new-launcher.png)
-
-Click on the **Terminal** option under **Other** to open a Linux terminal:
+1. To submit your assignment (for example, named: **Assignment1.ipynb**) to the **assignments** directory, open the JuypterLab new launcher by clicking the **File** top menu option, and then selecting **New Launcher**.
 
-![Alt text](../images/cps-ood-jupyterlab-open-terminal.png)
+1. Click on the **Terminal** option under **Other** to open a Linux terminal.
 
-Navigate to your personal directory by typing the following command (change the class name from `ALY3070` to your class name accordingly):
+1. Navigate to your personal directory by typing the following command (change the class name from `ALY3070` to your class name accordingly).
 
-:::
-cd /work/cps/ALY3070/students/$USER
-:::
+    :::
+    cd /work/cps/ALY3070/students/$USER
+    :::
 
-Where `$USER` is a saved shell variable for your username. You can optionally also replace it with your username.
+1. Where `$USER` is a saved shell variable for your username. You can optionally also replace it with your username.
 
-Check that your assignment file is visible in the command line by typing `ls`. Then, Copy the assignment file to the **assignments** directory with this command (replace **Assignment1.ipynb** with your file name):
+1. Check that your assignment file is visible in the command line by typing `ls`. Then, copy the assignment file to the **assignments** directory with this command (replace **Assignment1.ipynb** with your file name).
 
-:::
-cp Assignment1.ipynb ../../assignments
-:::
+    :::
+    cp Assignment1.ipynb ../../assignments
+    :::
 
-To remove an existing assignment, type (replace **Assignment1.ipynb** with your file name):
-
-:::
-rm ../../assignments/Assignment1.ipynb
-:::
+1. To remove an existing assignment, type (replace **Assignment1.ipynb** with your file name).
 
-Close the Terminal tab when done.
+    :::
+    rm ../../assignments/Assignment1.ipynb
```

Comment:
Not certain of the file structure for submission of CPS assignments under the new file tree.